### DISCUSSION TITLE
Can I use a JSON script file?

### DISCUSSION THREAD

**Discussion by cuonghuunguyen** [artilleryio/artillery#1166] [Artillery]
As I said, I want to use JSON to write my script instead. It could help me writing objects, array more easily. Can I write my scripts by JSON?

**Comment by hassy** [comment]
hi @cuonghuunguyen 👋 yes, you can use JSON. For example, most of the scripts used in tests are in JSON:

https://github.com/artilleryio/artillery/tree/master/test/scripts
